### PR TITLE
Move code that is only used in the tests

### PR DESCRIPTION
**Pull request by jsturtevant**
This is some cleanup for functions that are only used in the tests for the shims.

### CODE REVIEW

File: crates/containerd-shim-wasmedge/src/instance.rs [hunk 1]
```diff
@@ -293,6 +276,24 @@ mod wasitest {
...
 
+    static mut STDIN_FD: Option<RawFd> = None;
+    static mut STDOUT_FD: Option<RawFd> = None;
+    static mut STDERR_FD: Option<RawFd> = None;
+
+    fn reset_stdio() {
+        unsafe {
+            if STDIN_FD.is_some() {
+                dup2(STDIN_FD.unwrap(), STDIN_FILENO);
+            }
```
Comment (utam0k) [inline crates/containerd-shim-wasmedge/src/instance.rs:287]:
```suggestion
            if let Some(fd) = STDIN_FD {
                dup2(fd, STDIN_FILENO);
            }
```

**Review by Mossaka — APPROVED** [review]
LGTM!

File: crates/containerd-shim-wasmedge/src/instance.rs (hunk 1 above)
Comment (Mossaka) [reply crates/containerd-shim-wasmedge/src/instance.rs]: Done.

**Comment by Mossaka** [comment]
Hey, I went ahead and merged main branch. After the CI passes, I will merge this in

**Review by cpuguy83 — APPROVED** [review]

**Pull request merged**